osu.Server.Spectator/Hubs/Metadata/MetadataHub.cs

```diff
@@ -222,7 +238,11 @@ private Task broadcastUserPresenceUpdate(int userId, UserPresence? userPresence)
             // we never want appearing offline users to have their status broadcast to other clients.
             Debug.Assert(userPresence?.Status != UserStatus.Offline);
 
-            return Clients.Group(ONLINE_PRESENCE_WATCHERS_GROUP).UserPresenceUpdated(userId, userPresence);
+            return Task.WhenAll
+            (
+                Clients.Group(ONLINE_PRESENCE_WATCHERS_GROUP).UserPresenceUpdated(userId, userPresence),
+                Clients.Group(FRIEND_PRESENCE_WATCHERS_GROUP(userId)).FriendPresenceUpdated(userId, userPresence)
```

Comment:
I guess clients are going to get double status updates for friends, but probably fine for now.